src/EA.Weee.Web/Areas/Scheme/Views/ManageEvidenceNotes/_ManageEvidenceNotesHeaderAndTabs.cshtml

```diff
@@ -0,0 +1,30 @@
+﻿@using EA.Weee.Web.Areas.Scheme.Controllers
+@using EA.Weee.Web.Areas.Scheme.ViewModels.ManageEvidenceNotes
+@using EA.Weee.Web.RazorHelpers
+@using MvcNavigationHelpers
+@using HomeController = EA.Weee.Web.Areas.Scheme.Controllers.HomeController
+
+@model ManageEvidenceNoteViewModel
+@{
+    var title = String.Format("Manage evidence notes for {0}", Model.OrganisationName);
+    ViewBag.Title = title + " - GOV.UK";
+
+}
+
+<p>
+    @(this.WeeeGds().BackLink(@Url.UrlFor<HomeController>(a => a.ChooseActivity(Model.OrganisationId))))
+</p>
+
+<header class="hgroup">
+    <h1 class="govuk-heading-l">
+        @title
+    </h1>
+</header>
+
+<ul class="govuk-tabs__list">
+    @this.WeeeGds().Tab("Review submitted evidence",
+                        Url.UrlFor<ManageEvidenceNotesController>(a => a.Index(Model.OrganisationId, ManageEvidenceNotesDisplayOptions.ReviewSubmittedEvidence)),
```

Comment:
![image](https://user-images.githubusercontent.com/43237115/162402522-e3b6756d-e202-46d7-aead-c480e30a6361.png)

For some reason even though single tab it is showing as deselected. Have a look at the AATF tabs to see if the difference

Reply:
Fixed by adding default view option to the model (same way Meg has it with hers) - Future tab view models will need to do the same, also maybe with some logic in the controller itself